model/dce-manager.cc

```diff
@@ -400,17 +402,9 @@ DceManager::Start (std::string name, std::string stdinfilename, std::vector<std:
                    uid_t uid, uid_t euid, uid_t gid, uid_t egid)
```

Comment:
You can just remove this function and modify the prortype of the other function to `TaskManager::Start (void (*fn)(void*), void *context, uint32_t stackSize = 0)`. Please add a comment referring the ns3::Attribute value to which the value will be initialized when set to 0.